src/manifest.rs

```diff
@@ -1,3 +1,5 @@
+use anyhow::Context as _;
```

Comment:
Do we need the `as _` bit here?

Reply:
Done in 3b18ce47cc9e14eaa9e63281968e8dc9eaf3420b. I was basing this code on `manifest.rs` which also had `as _`, so I removed it there also.

Reply:
Ah I think I just realized what was going on here. `Context` is a trait, probably? And the `as _` means that the trait's methods are brought into scope without bringing the trait itself into scope.